.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-dist
\ No newline at end of file
+dist
+*lock
```

Comment:
Is this avoid the package-lock.JSON from being committed?

Reply:
yes to avoid tracking those changes at all